components/lib/splitbutton/splitbutton.d.ts

```diff
@@ -239,6 +239,11 @@ export interface SplitButtonProps extends Omit<React.DetailedHTMLProps<React.HTM
      * @defaultValue false
      */
     unstyled?: boolean;
+    /**
+     * Specifies if pressing escape key should hide the Split button Popup.
+     * @defaultValue true
+     */
+    closeOnEscape?: boolean | undefined;
```

Comment:
Move this up to be in alphabetical order

Reply:
@melloware But I see other props are not sorted alphabetically! Should I sort the entire list?